src/sparseml/pytorch/torchvision/train.py

```diff
@@ -430,6 +454,15 @@ def collate_fn(batch):
         if scaler:
             scaler.load_state_dict(checkpoint["scaler"])
 
+        # NOTE: override manager with the checkpoint's manager
+        manager = ScheduledModifierManager.from_yaml(checkpoint["checkpoint_recipe"])
+        checkpoint_manager = None
+
+        # TODO do we still do this?
+        manager.apply_structure(model_without_ddp, epoch=checkpoint["epoch"])
```

Comment:
@KSGulin if resuming, do we still do this apply_structure since the model is loaded above? And also how do checkpoint_path/resume affect the manager.modify below - does that only happen if we are NOT doing either resume/checkpoint_path?